### ISSUE TITLE
Using TensorFlowLiteSwift in SPM causes code signing issues

### ISSUE THREAD

**Issue by tomekfab** [kewlbear/TensorFlowLiteSwift#6]
I tried adding TensorFlowLiteSwift via SPM to our project, beginning a slow transition from carthage, but each time I add this particular lib I'm getting code signing issues and we can run app on device from Xcode

![Screenshot 2021-09-30 at 09 00 30](https://user-images.githubusercontent.com/39048175/135403200-e82ae686-617d-471f-aa35-bd533d455b8d.png)

**Comment by kewlbear** [comment]
Have you seen this? https://developer.apple.com/forums/thread/679182

**Comment by tomekfab** [comment]
Yup, I did. I've spent a good amount of time trying to research and resolve this before I created this issue. I've tried adding other dependencies to our project via SPM and none of the other caused this problem except this one, I'm not very fluent in SPM since this is the first time I use it, but I was looking into `artifacts` in Derived Data and saw that when adding `TensorFlowLiteSwift` to our project a `TensorFlowLiteC.xcframework` is created in `artifacts`, when I added it directly to our project it was automatically added with option `Embed & Sign` and I had the same issue, only when I changed it to `Do Not Embed` I was able to run our app on device. Not sure if there's anything to make out of this information but I thought it might be relevant somehow

**Comment by kewlbear** [comment]
So the issue seems related to https://developer.apple.com/documentation/xcode/using-the-latest-code-signature-format.  I tried various workarounds but can not get `CodeDirectory v=20400` for TensorFlowLiteC frameworks.

**Comment by tabqwertz** [comment]
Any updates on this? Thanks!

**Comment by ardyanrizki** [comment]
I got a similar issue. Any updates?

**Comment by tareksabry1337** [comment]
@kewlbear I was able to fix the code-signing shenanigans in TensorFlowLiteC / TensorFlowLiteCCoreML, I have created a repository that has SPM support for TensorFlowLiteC / TensorFlowLiteCCoreML, here's the link for [that](https://github.com/tareksabry1337/TensorFlowLiteC) I'm eventually going to create TensorFlowLiteSwift as well that mirrors the current releases on TensorFlowLiteC, but one step at time.

**Comment by tareksabry1337** [comment]
I have TesnorFlowLiteSwift out, Check it out using this [link](https://github.com/tareksabry1337/TensorFlowLiteSwift), It shouldn't have any code signing issues
